Misc/NEWS.d/next/Build/2023-01-21-10-31-35.gh-issue-101152.xvM8pL.rst

```diff
@@ -0,0 +1,2 @@
+Added the PEP-699 Implementation, :meth:`ma_version_tag` is deprecated for extension modules you can continue using it for the core code.
+The deprecation warning will pop up for extension modules.
```

Comment:
Builtin CPython code should not use the field either. We shouldn't allow this by saying you "can continue using it".

I suggest the following,

```suggestion
In accordance with PEP 699, the ``ma_version_tag`` field in :c:type:`PyDictObject`
is deprecated for extension modules. Accessing this field will generate a compiler
warning at compile time. This field will be removed in Python 3.13.
```